floss/main.py

```diff
@@ -253,11 +254,32 @@ def make_parser(argv):
     logging_group.add_argument(
         "-q", "--quiet", action="store_true", help="disable all status output on STDOUT except fatal errors"
     )
+    logging_group.add_argument(
+        "--color",
+        type=str,
+        choices=("auto", "always", "never"),
+        default="auto",
+        help="enable ANSI color codes in results, default: only during interactive session",
+    )
 
     return parser
 
 
-def set_log_config(debug, quiet):
+def set_log_config(debug, quiet, color):
+
+    if color == "always":
+        colorama.init(strip=False)
+    elif color == "auto":
+        # colorama will detect:
+        #  - when on Windows console, and fixup coloring, and
+        #  - when not an interactive session, and disable coloring
+        # renderers should use coloring and assume it will be stripped out if necessary.
+        colorama.init()
+    elif color == "never":
+        colorama.init(strip=True)
+    else:
+        raise RuntimeError("unexpected --color value: " + color)
```

Comment:
would you move this into main, under the call to `set_log_config`? the coloring configuration isn't really part of logging (though it *does* have to do with program output, so there's some similarity) so i don't think it fits here.

alternatively, you could introduce a new function like `set_color_config` to hold this logic.